Airports/EGAB/SMR/regions.txt

```diff
@@ -0,0 +1,624 @@
+N054.24.10.220 W007.39.21.825
+N054.24.10.267 W007.39.22.362
+N054.24.10.254 W007.39.22.806
+N054.24.10.226 W007.39.23.186
+N054.24.10.131 W007.39.23.413
+N054.24.09.997 W007.39.23.740
+N054.24.09.846 W007.39.24.104
+N054.24.09.635 W007.39.24.488
+N054.24.09.414 W007.39.24.831
+N054.24.09.166 W007.39.25.265
+N054.24.08.373 W007.39.24.115
+; - Grass
+REGIONNAME St Angelo
+grass N054.23.50.367 W007.38.57.444
+N054.23.49.871 W007.38.57.175
+N054.23.45.146 W007.38.50.372
+N054.23.45.265 W007.38.49.363
+N054.23.45.430 W007.38.48.934
+N054.23.45.683 W007.38.48.436
+N054.23.46.038 W007.38.47.852
+N054.23.46.302 W007.38.47.420
+N054.23.46.714 W007.38.47.387
+N054.23.51.188 W007.38.53.954
+N054.23.51.812 W007.38.54.912
+N054.23.52.276 W007.38.55.747
+N054.23.50.816 W007.38.57.539
+; - Road
+REGIONNAME St Angelo
+smrRoad N054.23.46.056 W007.39.01.633
+N054.23.49.686 W007.38.43.833
+N054.23.49.801 W007.38.43.883
+N054.23.46.179 W007.39.01.740
+; - Grass
+REGIONNAME St Angelo
+grass N054.23.41.391 W007.38.43.486
+N054.23.41.405 W007.38.43.050
+N054.23.41.469 W007.38.42.233
+N054.23.41.686 W007.38.41.906
+N054.23.43.092 W007.38.39.693
+N054.23.43.407 W007.38.39.592
+N054.23.43.818 W007.38.39.670
+N054.23.44.256 W007.38.39.871
+N054.23.44.680 W007.38.40.187
+N054.23.45.118 W007.38.40.446
+N054.23.45.293 W007.38.40.875
+N054.23.45.667 W007.38.41.559
+N054.23.46.810 W007.38.43.543
+N054.23.46.453 W007.38.45.558
+N054.23.46.401 W007.38.45.738
+N054.23.45.131 W007.38.48.215
+N054.23.45.045 W007.38.48.374
+N054.23.44.753 W007.38.48.796
+N054.23.44.113 W007.38.48.841
+N054.23.43.771 W007.38.48.332
+N054.23.41.546 W007.38.45.088
+; - Apron
+REGIONNAME St Angelo
+smrDarkApron N054.23.44.118 W007.38.48.853
+N054.23.44.750 W007.38.48.790
+N054.23.45.096 W007.38.48.277
+N054.23.45.347 W007.38.47.802
+N054.23.45.552 W007.38.47.381
+N054.23.46.411 W007.38.45.737
+N054.23.46.817 W007.38.43.550
+N054.23.48.126 W007.38.45.903
+N054.23.47.782 W007.38.46.555
+N054.23.47.870 W007.38.46.686
+N054.23.47.537 W007.38.47.525
+N054.23.48.340 W007.38.48.779
+N054.23.48.070 W007.38.49.384
+N054.23.47.767 W007.38.48.922
+N054.23.46.715 W007.38.47.379
+N054.23.46.308 W007.38.47.431
+N054.23.45.726 W007.38.48.363
+N054.23.45.257 W007.38.49.353
+N054.23.45.141 W007.38.50.334
+; - Building
+REGIONNAME St Angelo
+smrTEBuilding N054.23.45.104 W007.38.39.146
+N054.23.45.017 W007.38.39.665
+N054.23.44.418 W007.38.39.420
+N054.23.44.317 W007.38.39.888
+N054.23.43.826 W007.38.39.675
+N054.23.43.923 W007.38.38.612
+; - Building
+REGIONNAME St Angelo
+smrTEBuilding N054.23.40.971 W007.38.37.550
+N054.23.41.303 W007.38.37.032
+N054.23.41.891 W007.38.38.052
+N054.23.41.529 W007.38.38.584
+; - Grass
+REGIONNAME St Angelo
+grass N054.24.02.516 W007.39.22.367
+N054.24.02.658 W007.39.22.176
+N054.24.02.780 W007.39.22.321
+N054.24.02.809 W007.39.22.785
+N054.24.02.666 W007.39.22.991
+N054.24.02.469 W007.39.22.734
+; - Untitled_Polygon_2
+REGIONNAME St Angelo
+smrRoad 43605 N054.24.02.012 W007.39.22.072

```

Comment:
Here, the number 43605 needs to be removed/changed

Reply:
That should be that all tidied up there now, cheers for letting me know